tests/pass/miri_start.rs

```diff
@@ -1,4 +1,5 @@
 //@compile-flags: -Cpanic=abort
+//@only-miri: using miri internal apis
```

Comment:
```suggestion
//@only-miri: using internal APIs
```
Happens a few more times -- please use a consistent string here. Maybe `using Miri APIs` would be better, actually?